src/App.tsx

```diff
@@ -83,7 +85,7 @@ const App = () => {
 
     const theme = { ...baseTheme, colors: themeModes[themeMode] };
 
-    useBackgroundEventHandler();
+    const runEventHandlers = useBackgroundEventHandler();
```

Comment:
We cannot move the background event listener handler deep down to the React tree, because we may loose some events due to late event listener registration:


```javascript
        runtime.onMessage.addListener(function (request) {
            switch (request.type) {
                case 'CONNECT_UI': {
                    setEvent
```